### DISCUSSION TITLE
How can I add images to any page or post?

### DISCUSSION THREAD

**Discussion by diegoalarc** [MrGreensWorkshop/MrGreen-JekyllTheme#98] [How to] [answered]
This is my web https://diegoalarc.github.io/tabs/about.html

I would like to include images but I can't do that in the traditional markdown way.

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi, check this page
https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/_posts/2022-02-10-Examples.markdown

These are common for all pages and blog posts.

### Left aligned picture

`![such a lovely place](:hello.jpg)`

### Centered picture

`![such a lovely place](:hello.jpg){:data-align="center"}`

### Picture paths
for picture paths you have 2 options.
1. using auto path adder (":" is automatically replaced with `/assets/img/<layout>/`)
 - markdown: `![such a lovely place](:hello.jpg)` (example for any page)
 - pic path: `/assets/img/about/hello.jpg`  

2. using absolute paths
 - markdown:`![such a lovely place](/assets/img/about/hello.jpg)` (example for about page)
 - pic path: `/assets/img/about/hello.jpg`

### optional features
- to enable image viewer, uncomment this line 
image_viewer_on: true
- to enable image lazy loader, uncomment this line 
image_lazy_loader_on: true

**Comment by diegoalarc** [comment]
Now is working! Thanks for the help.
